packages/ckeditor5-link/docs/_snippets/features/link.js

```diff
@@ -35,6 +35,26 @@ ClassicEditor
 			allowExternalImagesEditing: [ /^data:/, 'origin', /ckbox/ ],
 			forceDemoLabel: true
 		},
+		link: {
+			addTargetToExternalLinks: false,
+			decorators: [
+				{
+					mode: 'manual',
+					label: 'Downloadable',
+					attributes: {
+						download: 'download'
+					}
+				},
+				{
+					mode: 'manual',
+					label: 'Open in a new tab',
+					attributes: {
+						target: '_blank',
+						rel: 'noopener noreferrer'
+					}
+				}
```

Comment:
I didn't add the gallery link property I saw in other demos because it may confuse users, I think.

Reply:
Good.